recipes/alps/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "alps" %}
+{% set version = "2.3.b1" %}
+{% set revision = "r7782" %}
+{% set sha256 = "69910b4fcd5e98a01741843de530f896864a390f693bf010b2db56ceb3f071d7" %}
+{% set mpi = os.environ.get('MPI') or 'mpich' %}
+{% set blas = os.environ.get('BLAS') or 'openblas' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  url: http://alps.comp-phys.org/static/software/releases/{{ name }}-{{ version }}-{{revision}}-src.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win]
+  skip: true  # [not py27]
+  number: 0
+  detect_binary_files_with_prefix: True
+  features:
+    - blas_{{blas}}
+
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - icu 56.*  # [unix]
+    - boost
+    - python
+    - numpy
+    - scipy
+    - matplotlib
+    - hdf5
+    - blas 1.* {{blas}}
+    - {{mpi}}
```

Comment:
I actually have to remove ```mpi``` because for the moment ```boost``` is built without.

The others are all needed during build. ```matplotlib``` dependency will be removed in the next release, now unfortunately there is a test which loads it for no reason.

let me actually submit a patch to remove the two dependencies.